Lib/http/server.py

```diff
@@ -562,6 +563,10 @@ def log_error(self, format, *args):
 
         self.log_message(format, *args)
 
+    # https://en.wikipedia.org/wiki/List_of_Unicode_characters#Control_codes
+    __control_char_table = str.maketrans(
```

Comment:
Why __ instead of _?

Reply:
A habit for backports to make it hard for someone to rely on as an API and avoid conflicts.  realistically it shouldn't matter, want a single _?